app/views/shelters/show.html.erb

```diff
@@ -23,3 +23,17 @@
     <%= link_to "Pets", "/shelters/#{@shelter.id}/pets", class: "card-footer-item" %>
   </footer>
 </div>
+
+<div class="reviews">
```

Comment:
Smart to put the reviews within their own div class. I'm trying to work on using tags like this more in how I write my original view to save time styling later.